src/test/regress/expected/multi_create_table.out

```diff
@@ -650,8 +653,7 @@ COMMIT;
 \c - - - :master_port
 DROP TABLE tt1;
 DROP TABLE tt2;
--- It is known that creating a table with master_create_empty_shard is not
--- transactional operation.
+-- It is known that creating a table with master_create_empty_shard is transactional
```

Comment:
Is it? I thought we only made `create_distributed_table` transactional. `master_create_empty_shard` has a bunch of legacy applications, so it might be better to keep it non-transactional.